client/my-sites/comments/comment-threaded-list/threaded-comment.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { filter, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Comment from 'my-sites/comments/comment';
+import Notice from 'components/notice';
+
+export class ThreadedComment extends Component {
+	static propTypes = {
+		commentId: PropTypes.number,
+		commentsTree: PropTypes.array,
+		depth: PropTypes.number,
+	};
+
+	state = {
+		isExpanded: false,
+	};
+
+	showReplies = () => this.setState( { isExpanded: true } );
+
+	render() {
+		const { commentId, commentsTree, depth, translate } = this.props;
+		const { isExpanded } = this.state;
+
+		const replyTree = filter( commentsTree, { commentParentId: commentId } );
+
+		return (
+			<div className={ `comment-threaded-list__comment depth-${ depth }` }>
+				<Comment { ...{ commentId } } isPostView refreshCommentData />
+
+				{ !! replyTree.length &&
+					! isExpanded && (
+						<Notice
+							className="comment-threaded-list__more-replies"
+							icon="reply"
+							showDismiss={ false }
+						>
+							<a onClick={ this.showReplies }>{ translate( 'Show replies' ) }</a>
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Maximum replies to show:' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).